api-gateway/src/main/resources/static/scripts/owner-details/owner-details.controller.js

```diff
@@ -8,11 +8,11 @@ angular.module('ownerDetails')
             self.owner = resp.data;
         });
 
-        $http.update('api/gateway/owners/' + $stateParams.ownerId).then(function (resp) {
+        /*$http.update('api/gateway/owners/' + $stateParams.ownerId).then(function (resp) {
             self.owner = resp.data;
         });
 
         $http.delete('api/gateway/owners/' + $stateParams.ownerId).then(function (resp) {
             self.owner = resp.data;
-        });
+        });*/
```

Comment:
I'm not sure I understand why these lines might be the ones causing the issue. We need to make sure that we still keep in order to call an update/delete HTTP request!